conda_testenv/tests/integration/test_recipes/b/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+    name: b
+    version: 1.2
+
+requirements:
+    build:
+        - python
+    run:
+        - python
```

Comment:
Perhaps add a ``numpy x.x`` into the build and run deps here, to exercise that CONDA_NPY line.